### ISSUE TITLE
list of tnsors visualzation fails with error "Can't call numpy() on Tensor that requires grad. Use tensor.detach().numpy() instead."

### ISSUE THREAD

**Issue by kapulkin** [microsoft/vscode-data-wrangler#267]
Type: <b>Bug</b>

# Behaviour

## Expected vs. Actual

List of tensors should be visualized but it fails

## Steps to reproduce:

1. Run
output = model.generate(**tokens, max_length=10, output_hidden_states=True, return_dict_in_generate=True, output_logits=True)
for any model in interactive mode and load logits into separate variable
logits = [lm_head(h[-1]) for h in output.hidden_states]

2. Try to view logits vairable

# Diagnostic data

-   Jupyter extension version: 2024.7.0
-   Python extension version: 2024.12.3
-   .NET Install Tool for Extension Authors extension version: Not installed
-   Python package dependencies:

```json
{
  "installed": {
    "pandas": "2.2.2",
    "pyspark": "3.5.1",
    "pyarrow": "15.0.2"
  },
  "required": {
    "pandas": "1.2.0"
  },
  "unsatisfied": []
}
```

-   Entrypoint: JupyterDataViewer
-   Active mode: dataViewer

Extension version: 1.6.0
VS Code version: Code 1.92.2 (fee1edb8d6d72a0ddff41e5f71a671c23ed924b9, 2024-08-14T17:29:30.058Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 x 2808)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.86GB (0.46GB free)|
|Process Argv|--crash-reporter-id 88348fd0-af71-4d43-98a0-68b8576a6fbb|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonnoceb:30805159
asynctok:30898717
pythonregdiag2:30936856
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
0ee40948:31013168
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
dwnewjupytercf:31046870
newcmakeconfigv2:31071590
impr_priority:31102340
refactort:31108082
pythonrstrctxt:31112756
flightc:31119335
wkspc-onlycs-c:31111717
wkspc-ranged-c:31123312
ei213698:31121563

```

</details>

**Comment by pwang347** [comment]
Hi @kapulkin, thank you for the patience. Wanted to let you know that this scenario should no longer cause a failure as of pre-release 1.13.2. Please feel free to give it a try and let us know if you have any additional issues with this. Thanks!
